zds/utils/management/commands/migrate_from_gravatar.py

```diff
@@ -0,0 +1,26 @@
+from hashlib import md5
+from time import sleep
+
+import requests
+from django.core.management.base import BaseCommand
+
+from zds.member.models import Profile
+
+
+class Command(BaseCommand):
+    help = "Migrate from Gravatar"
+
+    def handle(self, *args, **options):
+        total = Profile.objects.filter(avatar_url__isnull=True).count()
+        i = 1
+        for profile in Profile.objects.select_related("user").filter(avatar_url__isnull=True).iterator():
```

Comment:
En fait on a des champs `avatar_url` qui sont NULL et des champs qui contiennent une chaîne de caractères vide.

Donc il faut modifier : 
```suggestion
        # We have profiles with either NULL or empty avatar_url field
        for profile in Profile.objects.select_related("user").filter(Q(avatar_url__isnull=True) | Q(avatar_url="")).iterator():
```

Il faut aussi faire la modification dans la requête qui compte combien il y a de profils à traiter, et ajouter : 
```py
from django.db.models import Q
```

Et je me faisais la réflexion que cette commande aurait plutôt sa place dans le module `member` que `utils`, non ?